corehq/form_processor/document_stores.py

```diff
@@ -86,12 +86,8 @@ def product_ids(self):
         return Product.ids_by_domain(self.domain)
 
     def iter_document_ids(self, last_id=None):
-        from corehq.form_processor.parsers.ledgers.helpers import UniqueLedgerReference
-        # todo: support last_id
-        # assuming we're only interested in the 'stock' section for now
-        for case_id in self.case_accessors.get_case_ids_in_domain(self.domain):
-            for product_id in self.product_ids:
-                yield UniqueLedgerReference(case_id, 'stock', product_id).to_id()
```

Comment:
how was this even working?
the method name is `as_id` [here](https://github.com/dimagi/commcare-hq/blob/16ca9adba01ba6a7723d2a60699a62b85177314b/corehq/form_processor/parsers/ledgers/helpers.py#L16)

Reply:
yup

Reply:
interesting.